.env

```diff
@@ -0,0 +1,10 @@
+# This file is a "template" of which env vars need to be defined for your application
```

Comment:
should this file be committed? i think only the .env.dist should be, and .env should be in .gitignore

Reply:
you are completely right